docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -116,19 +116,31 @@ export default WysiwygWithErrors;
 
 #### Styles
 
-The AdminUI package source can be easily found in `./node_modules/strapi-admin/src/`.
+The AdminUI package source is located in `./node_modules/strapi-admin/admin/src/`.
 
-For example, to change the top-left displayed admin panel's color, copy the `./node_modules/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader` folder to `./admin/src/components/LeftMenu/LeftMenuHeader` (create these folders if they don't exist) and change the styles inside `./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js`.
+For example, to change the top-left admin panel's color:
 
+1. create a copy of the 
+- `./node_modules/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader` folder to 
+- `./admin/src/components/LeftMenu/LeftMenuHeader` (create these folders if they don't exist)
 
-Thus, you are replacing the files that would normally be in `node_modules/strapi-admin/admin/src` and directing them to `admin/src/some/file/path`.
+```
+mkdir -p ./admin/src/components/LeftMenu/LeftMenuHeader
+cp ./node_modules/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js ./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js
+```
 
-To apply your changes you need to rebuild your admin panel
+2. modify style within that copy (i.e. `background-color: #007eff;`, also see [sourcecode online](https://github.com/strapi/strapi/blob/master/packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js#L7)):
 
 ```
-npm run build
+vi ./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js
```

Comment:
I'm a huge fan of vi myself, but I'm just wondering:
* is vi installed on every system by default?
* isn't there a more generic way to edit files? 

We're trying to have our documentation as inclusive as possible.
I wouldn't like people to be "unable to exit vi" :D 

I would suggest something like:

```suggestion
2. modify style within that copy (i.e. `background-color: #007eff;`, also see [sourcecode online](https://github.com/strapi/strapi/blob/master/packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js#L7)): open and edit `./admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js` with your favorite editor. 

```

Reply:
We shouldn't make assumptions about editor, what we have done in the past is provide a path to the file like:

```
**Path:** `./path/to/the/file`
```

Then provide the changes needed in a code block marking the language. Editors vary far too much from system to system, specifically windows.

Reply:
Very valid comment, thanks. Getting used to Open Source and infinite variety of possible ways for anything to go wrong :)